dictionary.txt

```diff
@@ -0,0 +1,170 @@
+1000s
+1rem
+ACR
+APIs
+ARNs
+AWS:SourceArn
+Api
+ArnLike
+Auth
+BoltDB
+CDK
+CLI
+CLI's
+CORS
+CloudRun
+CloudWatch
+Config
+Dev
+DigitalOcean
+ECR
+ExpressJS
+FireStore
+GCP
+Homebrew
+HttpMiddleware
+IAM
+JSON
+macOS
+Microservice
+Middleware
+MinIO
+Multi-Repo
+Multi-Repos
+NPM
+OCI
+ORM
+ORMs
+OpenAPI
+PARAM_ID
+PubSub
+Repo
+S3
+SDK
+SDKs
+SNS
+SQS
+SaaS
+Serverless
+SpringBoot
+Stackoverflow
+UUID
+Uint8Array
+Vendia
+api
+apiV1
+apis
+args
+arn
+async
+aws
+aws_access_key_id
+aws_secret_access_key
+azure-cli
+backend
+batchQueue
+batchQueue.send
+br
+br/
+cd
+codebase
+config
+config-as-code
+const
+create-a-route-with-path-params
+createCustomer
+cron
+css
+ctrl
+ctrl-C
+ctx
+customerData
+customerId
+customerRoute
+customerRoute.delete
+customerRoute.get
+customerRoute.patch
+customerRoute.post
+customerRoute.put
+customersHandler
+customersRoute
+customersRoute.all
+decrypt
+deleteCustomer
+dev
+eventing
+fasthttp
+filepath
+flexDirection
+gRPC
+gcp
+getAllCustomers
+getCustomer
+gridAutoRows
+helloApi
+href
+http
+javascript
+keyRef
+keyVersionRef
+koa
+lambda:InvokeFunction
+latestVersion
+latestVersionRef
+localhost:9001
+matcher
+microservices
+middleware
+newApiKeyVersionRef
+nitric-apis
+nitrictech
+npm
+openAPI
+orderTopic
+package.json
+pagingToken
+param
+params
+patchCustomer
+payloadType
+persistance
+postCustomer
+pre-built
+pre-compiled
+pre-signed
+privateApi
+publicApi
+putCustomer
+repo
+responseBody
+roadmap
+rowGap
+serverless
+sudo
+toml
+triages
+txt
+undeploy
+url
+v1
+v1/
+v2
+v3
+versionId
+x-amazon-apigateway-integration
+yaml
+Matchers
+deployable
+scalable
+unhandled
+queryable
+composable
+integrations
+Scalable
+Transactional
+Pulumi
+homebrew
+PubSubPull
+\..+
```

Comment:
can we use regex's in some of these cases?
e.g. br and br/ and other markup could be ignored using a standard html tag regex?

Reply:
Yes we can, this dictionary was auto generated by the library.

Reply:
@tjholm this is more for unknown words in the en-US dictionary. You can use regex for words, like the one for any with a dot in front of it.